metadata-ingestion/src/datahub/ingestion/source/gc/datahub_gc.py

```diff
@@ -292,6 +292,7 @@ def revoke_expired_tokens(self) -> None:
             tokens = list_access_tokens.get("tokens", [])
             total = list_access_tokens.get("total", 0)
             if tokens == []:
+                # Due to a server bug we cannot rely on just total
```

Comment:
how is this related to the `total` considering the if is about tokens: `if tokens == []`? 🤔

Reply:
I remembered that I had added this odd logic but forgot to add comment.